docs/links/knowledge_base_sf.py

```diff
@@ -0,0 +1,7 @@
+from . import link
+
+link_name = "Knowledgebase SF" 
+link_text = "Knowledgebase SF" 
```

Comment:
```suggestion
link_text = "Knowledgebase article on Salesforce" 
```

I think it'd read better for the user if the text was a little bit more descriptive - otherwise good to go!